scripts/dts/gen_defines.py

```diff
@@ -247,7 +247,7 @@ def write_props(node):
 def write_bus(node):
     # Generate bus-related #defines
 
-    if not node.bus:
+    if not node.on_bus:
         return
 
     if node.parent.label is None:
```

Comment:
we can rework this so its just:

```
if not node.bus_node:
   return
```

and than change `if node.parent.label is None:` to `if node.bus_node.label is None`

Reply:
Done. Thought `if node.bus_node` might be misread as "is the node a bus node?", but maybe it's clear enough.